### PR TITLE
kubevirt: migrate `getBootableDevicesInOrder()` and `BootOrder` from web-ui-components

### PR DESCRIPTION
**Pull request by mareklibra**
Another dependency less.
This is part of the gradual effort to remove the kubevirt-web-ui dependency.

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/src/selectors/vm/devices.ts [hunk 1]
```diff
@@ -1,5 +1,32 @@
+import { VMKind, V1NetworkInterface, VMLikeEntityKind } from '../../types';
+import { DEVICE_TYPE_DISK, DEVICE_TYPE_INTERFACE } from '../../constants';
+import { V1Disk } from '../../types/vm/disk/V1Disk';
+import { getDisks, getInterfaces } from './selectors';
+import { asVM } from './vmlike';
+
+type BootableDeviceType = {
+  type: string;
+  value: V1Disk | V1NetworkInterface;
+};
```
Comment (pcbailey) [inline frontend/packages/kubevirt-plugin/src/selectors/vm/devices.ts]: nitpick: I was told that we should declare all types at the bottom of the file. I'm not sure if this is actually an established convention, though.

**Comment by pcbailey** [comment]
I just had one small nitpick. Just ignore it if it's not an established convention. Other than that, I'm good with it.

/lgtm

File: frontend/packages/kubevirt-plugin/src/selectors/vm/devices.ts (hunk 1 above)
Comment (mareklibra) [reply frontend/packages/kubevirt-plugin/src/selectors/vm/devices.ts]: good point :-) I am stacking other work on top of this and moving the type to another file completely

**Comment by mareklibra** [comment]
Sorry for confusion, I am about to add another commit here
/lgtm cancel

File: frontend/packages/kubevirt-plugin/src/selectors/vm/devices.ts (hunk 1 above)
Comment (pcbailey) [reply frontend/packages/kubevirt-plugin/src/selectors/vm/devices.ts]: Ah! Carry on, then. =) I didn't notice the WIP in the title. Sorry for the added noise.

**Pull request merged**